src_c/event.c

```diff
@@ -1253,25 +1253,20 @@ dict_from_event(SDL_Event *event)
         case PGE_WINDOWDISPLAYCHANGED:
         case SDL_TEXTEDITING:
         case SDL_TEXTINPUT:
+        case SDL_DROPTEXT:
+        case SDL_DROPFILE:
```

Comment:
I'm fairly certain that adding only these two lines is all you had to do.

Reply:
![image](https://user-images.githubusercontent.com/31395137/202138762-30a32132-9a89-4940-8b7c-910497c84bb7.png)  
If this if statement is not added, the "window" attribute will be always "None".
![image](https://user-images.githubusercontent.com/31395137/202139737-e7905504-eb2d-46b1-b965-768e54807b03.png)

Reply:
@Matiiss

Reply:
Yeah, I understand, well, I'm not qualified enough to evaluate this, sorry, besides it would just be a suggestion anyways.